src/Akka.Cluster.Hosting/ClusterClientOptions.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Akka.Actor;
+using Akka.Cluster.Tools.Client;
+using Akka.Remote;
+
+namespace Akka.Cluster.Hosting
+{
+    public class ClusterClientOptions
+    {
+        /// <summary>
+        /// Actor paths of the <see cref="ClusterReceptionist"/> actors on the servers (cluster nodes) that the client will try to contact initially.
+        /// </summary>
+        public HashSet<ActorPath> InitialContacts { get; set; }
+
+        /// <summary>
+        /// Interval at which the client retries to establish contact with one of ClusterReceptionist on the servers (cluster nodes)
+        /// </summary>
+        public TimeSpan? EstablishingGetContactsInterval { get; set; }
+
+        /// <summary>
+        /// Interval at which the client will ask the <see cref="ClusterReceptionist"/> for new contact points to be used for next reconnect.
+        /// </summary>
+        public TimeSpan? RefreshContactsInterval { get; set; }
+
+        /// <summary>
+        /// How often failure detection heartbeat messages for detection of failed connections should be sent.
+        /// </summary>
+        public TimeSpan? HeartbeatInterval { get; set; }
+
+        /// <summary>
+        /// Number of potentially lost/delayed heartbeats that will be accepted before considering it to be an anomaly. 
+        /// The ClusterClient is using the <see cref="DeadlineFailureDetector"/>, which will trigger if there are 
+        /// no heartbeats within the duration <see cref="HeartbeatInterval"/> + <see cref="AcceptableHeartbeatPause"/>.
+        /// </summary>
+        public TimeSpan? AcceptableHeartbeatPause { get; set; }
```

Comment:
So one piece of feedback on this class, take it or leave it - I don't usually expose _every_ option on here. Only the 20% that 80% of people need (i.e. the `InitialContacts`.)

Might be a good idea to pare that down just to avoid people thinking they have to set all of these. Advanced users can always supply the HOCON instead. What do you think?

Reply:
Sounds good to me, we can ditch the options object altogether in that case.